ogr/services/gitlab/pull_request.py

```diff
@@ -118,16 +119,50 @@ def create(
         source_branch: str,
         fork_username: str = None,
     ) -> "PullRequest":
-        mr = project.gitlab_repo.mergerequests.create(
-            {
-                "source_branch": source_branch,
-                "target_branch": target_branch,
-                "title": title,
-                "description": body,
-            }
-        )
+        repo = project.gitlab_repo
+        parameters = {
+            "source_branch": source_branch,
+            "target_branch": target_branch,
+            "title": title,
+            "description": body,
+        }
+
+        if project.is_fork and (
+            fork_username is None or fork_username == project.namespace
+        ):
+            parameters["target_project_id"] = project.parent.gitlab_repo.attributes[
+                "id"
+            ]
+        elif fork_username and fork_username != project.namespace:
+            other_project = GitlabPullRequest.__get_fork(
+                fork_username,
+                project.get_forks()
+                if project.parent is None
+                else project.parent.get_forks(),
+            )
+
+            parameters["target_project_id"] = other_project.gitlab_repo.attributes["id"]
+            if project.parent is None:
+                parameters["target_project_id"] = repo.attributes["id"]
```

Comment:
maybe use one variable `target_id: Optional[int]` and just assign it if has value at the end? Would be cleaner